### PR TITLE
Improvements to logs

### PR DESCRIPTION
**Pull request by roshanrajaratnam**
* This Pull request improves the readability of BLE logs.

### CODE REVIEW

File: core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattDescriptor.kt [hunk 1]
```diff
@@ -60,9 +61,9 @@ data class NativeBluetoothGattDescriptor(
     override fun toString(): String {
         return StringBuilder()
             .append("{")
-            .append("uuid: $uuid")
-            .append("permissions: $permissions")
-            .append("value: ${value.toDisplayString()}")
+            .append("uuid : $uuid ")
```
Comment (philips77) [inline core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattDescriptor.kt:64]: why no spaces here after initial "?

**Review by philips77 — CHANGES_REQUESTED** [review]

File: core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattDescriptor.kt (hunk 1 above)
Comment (roshanrajaratnam) [reply core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattDescriptor.kt]: Permissions object has spaces has already, you may check on logcat.

File: core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattCharacteristic.kt [hunk 2]
```diff
@@ -65,14 +65,14 @@ data class NativeBluetoothGattCharacteristic(
...
         return StringBuilder()
-            .append("{")
-            .append(" uuid : $uuid, ")
-            .append(" instanceId : $instanceId,")
-            .append(" permissions : $permissions,")
-            .append(" properties : $properties,")
-            .append(" writeType : $writeType,")
-            .append(" value : ${value.toDisplayString()},")
-            .append(" descriptors : $descriptors")
+            .append(" { ")
```
Comment (philips77) [inline core/src/main/java/no/nordicsemi/android/kotlin/ble/core/wrapper/NativeBluetoothGattCharacteristic.kt:68]: Initial space? There's no such in 2 methods below.

**Review by philips77 — CHANGES_REQUESTED** [review]

File: client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattService.kt [hunk 3]
```diff
@@ -61,7 +61,7 @@ data class ClientBleGattService internal constructor(
      */
     val uuid = service.uuid
 
-    val characteristics = service.characteristics.map {
+    private val characteristics = service.characteristics.map {
```
Comment (sylwester-zielinski) [inline client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattService.kt:64]: We have a feature where we displays all of the characteristics. It needs to be public.

**Review by sylwester-zielinski — CHANGES_REQUESTED** [review]

File: client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattService.kt (hunk 3 above)
Comment (roshanrajaratnam) [reply client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattService.kt:64]: Ok sure, Android studio was complaining that this was not in use. You may add a suppression for this warning :)

Comment (roshanrajaratnam) [reply client/src/main/java/no/nordicsemi/android/kotlin/ble/client/main/service/ClientBleGattService.kt]: Done

**Review by sylwester-zielinski — APPROVED** [review]
Amazing!

**Pull request merged**